### PR TITLE
[SMALLFIX] Small fixes in several classes

### PR DESCRIPTION
**Pull request by HorizonNet**
- Fixed documentation in OpenFileEntry
- Removed not needed cast in ResourcePoolTest
- Collapsed multiple catch statements into one in TieredBlockStore
- Used foreach instead of for in TieredBlockStoreTestUtils
- Fixed documentation in TtlIntervalRule

### CODE REVIEW

File: core/server/src/main/java/alluxio/worker/block/TieredBlockStore.java [hunk 1]
```diff
@@ -744,14 +739,9 @@ private MoveBlockResult moveBlockInternal(long sessionId, long blockId,
...
         // TODO(bin): Implement rollback scheme to recover from IO failures.
         mMetaManager.moveBlockMeta(srcBlockMeta, dstTempBlock);
-      } catch (BlockAlreadyExistsException e) {
-        throw Throwables.propagate(e); // we shall never reach here
-      } catch (BlockDoesNotExistException e) {
+      } catch (BlockAlreadyExistsException | BlockDoesNotExistException
+          | WorkerOutOfSpaceException e) {
         throw Throwables.propagate(e); // we shall never reach here
-      } catch (WorkerOutOfSpaceException e) {
-        // Only possible if session id gets cleaned between createBlockMetaInternal and
```
Comment (jsimsa) [inline core/server/src/main/java/alluxio/worker/block/TieredBlockStore.java]: this comment got lost

**Comment by jsimsa** [comment]
left one minor comment

**Comment by apc999** [comment]
after fixing the minor comment from Jiri, LGTM
thanks for keeping improving the code health!

**Comment by HorizonNet** [comment]
@jsimsa Good catch and thanks for the comment. I updated the PR accordingly.

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**